install_files/securedrop-config/DEBIAN/control

```diff
@@ -0,0 +1,10 @@
+Source: securedrop
+Section: web
+Priority: optional
+Maintainer: SecureDrop Team <securedrop@freedom.press>
+Homepage: https://securedrop.org
+Package: securedrop-config
+Version: 0.1.0+0.4.3
+Architecture: amd64
```

Comment:
It should be **all** instead of **amd64** because the package is not architecture dependent. 

https://www.debian.org/doc/debian-policy/index.html#architecture